packages/shared/src/components/auth/PostAuthBanner.tsx

```diff
@@ -1,11 +1,116 @@
 import React, { ReactElement } from 'react';
+import { useSearchParams } from 'next/navigation';
+import { useQuery } from '@tanstack/react-query';
+import classNames from 'classnames';
 import { feature } from '../../lib/featureManagement';
 import { checkIsBrowser, checkIsExtension, UserAgent } from '../../lib/func';
 import { AuthExtensionBanner } from './AuthExtensionBanner';
 import { AuthenticationBanner } from './AuthenticationBanner';
+import { getBasicUserInfo } from '../../graphql/users';
+import { OnboardingHeadline } from './OnboardingHeadline';
+import { sizeClasses } from '../ProfilePicture';
+import { Image, ImageType } from '../image/Image';
 import { useConditionalFeature } from '../../hooks';
+import { IconSize } from '../Icon';
+import {
+  getSocialReferrer,
+  socialCTA,
+  socialGradient,
+  socialIcon,
+  type SocialIconType,
+} from '../../lib/socialMedia';
+import { capitalize } from '../../lib/strings';
+import { useAuthContext } from '../../contexts/AuthContext';
+
+const geoToEmoji = (geo: string): string => {
+  return geo
+    .toUpperCase()
+    .split('')
+    .map((char) => String.fromCodePoint(char.charCodeAt(0) + 0x1f1a5))
+    .join('');
+};
+
+const GeoPersonalizedBanner = ({ geo }: { geo: string }): ReactElement => {
+  const emoji = geoToEmoji(geo);
+  const displayNames = new Intl.DisplayNames(['en'], { type: 'region' });
+  const country = displayNames.of(geo.toUpperCase());
+
+  return (
+    <AuthenticationBanner>
+      <span className="text-[3.5rem] leading-none">{emoji}</span>
```

Comment:
Rather use Typoghraph element here, it should be `typo-mega1` from design.

![Screenshot 2024-11-25 at 08 46 23](https://github.com/user-attachments/assets/34fc87c3-a642-48a4-b291-4ec6a0c1afb4)